server/utils/placements/transfers.ts

```diff
@@ -0,0 +1,39 @@
+import { SummaryList, TransferFormData } from '@approved-premises/ui'
+import { Cas1PremisesBasicSummary } from '@approved-premises/api'
+import { DateFormats } from '../dateUtils'
+
+export const allApprovedPremisesOptions = (approvedPremises: Array<Cas1PremisesBasicSummary>) => [
+  {
+    value: null,
+    text: 'Select an Approved Premises',
+  },
+  ...approvedPremises
+    .filter(premises => premises.supportsSpaceBookings)
+    .map(premises => ({
+      value: premises.id,
+      text: `${premises.name} (${premises.apArea.name})`,
+    })),
+]
+
+export const transferSummaryList = (formData: TransferFormData): SummaryList => {
+  return {
+    rows: [
+      {
+        key: { text: 'Date of transfer' },
```

Comment:
Suggest using `summaryListItem()` to tidy these rows.

Reply:
Done!